### ISSUE TITLE
Ability to import binary data as an editable format

### ISSUE THREAD

**Issue by yoshiyoshyosh** [Yellow-Dog-Man/Resonite-Issues#2876]
### Is your feature request related to a problem? Please describe.

Currently, the only way to import a file and access its contents in some editable way is by using a .txt file (or any `text/` document) and copying/sourcing the `Content` of the text in the UIX panel that pops up. There's another issue about being able to import it directly as a string (#1754), but that's irrelevant to this issue.

The thing is, when importing a file in this way, Resonite uses the default [StreamReader](https://learn.microsoft.com/en-us/dotnet/api/system.io.streamreader?view=net-8.0) class to read the contents of the file. The default reader assumes that the input is UTF-8. This is fine for text files, where 99.99% of text in the world is UTF-8 by now, but it is not fine for arbitrary binary data, where any invalid UTF-8 characters are mapped to `U+FFFC`, destroying the binary data.

### Describe the solution you'd like

Some way, whether a special file extension like `.protofluxbin` or as part of an import dialog, to import a file as a raw, editable binary data format. Of course, the most obvious form of this would be an unsigned byte array, which is blocked by #572. A second-class option would be treating the input as ASCII to convert to UTF16, which would preserve all the bytes as `char`s in a `string`, but this would be a "hacky temporary solution", which is long established by now not the way to implement things in FrooxEngine.

### Describe alternatives you've considered

None? Not sure about any alternatives for this specific feature request.

### Additional Context

This is greatly useful for projects like microcontrollers, binary file format parsing, etc.

This is more of a "tracking issue" for a specific request once collections are added.

### Requesters

yosh, wattlefoxxo
